stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/sync/internal/SyncManager.kt

```diff
@@ -110,6 +110,17 @@ internal class SyncManager(
         }
     }
 
+    /**
+     * Store the latest sync state of events. Sync state is updated whenever sync api returns a success response.
+     */
+    internal suspend fun updateSyncStateForEvents(latestEventDate: Date) {
```

Comment:
The method is responsible for updating only `lastSyncedAt`. What do you think about renaming it to `updateLastSyncedDate` or similar?